### PR TITLE
bugsnag order cancel exception

### PR DESCRIPTION
**Pull request by danac-gs**
# Description
Please include a summary of the change and which issue is fixed. Include the motivation for the changes, and comment on your PR if necessary for clarity.

Fixes: (link Asana Task)

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [x] Successfully tested on a staging or sandbox server
- [x] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes
- [ ] I have created or modified unit tests to sufficiently cover my changes.

### CODE REVIEW

File: Helper/Order.php [hunk 1]
```diff
@@ -878,6 +878,7 @@ protected function deleteOrder($order)
         try {
             $order->cancel()->save()->delete();
         } catch (\Exception $e) {
+            $this->bugsnag->notifyException($e);
```
Comment (YevhenShBolt) [inline Helper/Order.php]: @danac-gs   Do we need here some additional ex. order id or order increment_id?
As you think?

I thinking about situation when we will have an exception and additionally we already will have some additional info.

Comment (matt-thomason) [reply Helper/Order.php]: I think the exception message will have most of that information, but I could be mistaken.

Comment (danac-gs) [reply Helper/Order.php]: @YevhenShBolt IDs are added to log. Please approve.

**Comment by danac-gs** [comment]
@matt-thomason @ethanwayda22 can you check what's wrong with the integration tests and merge this pr?

**Pull request merged**